presto-function-namespace-managers/src/main/java/com/facebook/presto/functionNamespace/mysql/FunctionNamespaceDao.java

```diff
@@ -64,8 +66,8 @@
             "  description text,\n" +
             "  deleted boolean NOT NULL DEFAULT false,\n" +
             "  delete_time TIMESTAMP NULL DEFAULT NULL,\n" +
-            "  create_time TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,\n" +
-            "  update_time TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,\n" +
+            "  create_time TIMESTAMP NULL DEFAULT CURRENT_TIMESTAMP,\n" +
```

Comment:
Why not require these to be `NOT NULL`?

Reply:
Insert null to a non-null timestamp column is valid in MySQL, and MySQL stores `0000:00:00 00:00:00` into the column, which is a dangerous behavior and the MySQL team advices against that.